src/download.test.js

```diff
@@ -0,0 +1,33 @@
+const axios = require("axios");
+const tmp = require("tmp");
+
+const { downloadFile } = require("./download");
+
+jest.mock("axios");
+
+const portalUrl = "https://siasky.net";
+const skylink = "XABvi7JtJbQSMAcDwnUnmp2FKDPjg8_tTTFP4BwMSxVdEg";
+
+describe("download", () => {
+  const body = "asdf";
+
+  axios.get.mockResolvedValue({ data: { body, pipe: function () {} } });
```

Comment:
this should be in `beforeEach` block, otherwise every test in this `describe` block will share one instance of the mocked function and we want to run tests in isolation

```suggestion
  beforeEach(() => {
    axios.get.mockResolvedValue({ data: { body, pipe: function () {} } });
  });
```